### PR TITLE
controlled scrolling for representatives

### PR DESCRIPTION
**Pull request by sammychinedu2ky**
Before the changes I made, scrolling through the representative column removed the focus from the letter creation section. You can see it in the video below.

# Before
https://user-images.githubusercontent.com/36219292/201768866-a13ff604-2131-43d7-9534-587ed84f909d.mp4

# After my changes

https://user-images.githubusercontent.com/36219292/201768956-7b4aca25-c1dd-429e-98ab-14607b0620a3.mp4

@paramsiddharth 
@manishapriya94

### CODE REVIEW

**Review by manishapriya94 — APPROVED** [review]
Looks good!

**Comment by paramsiddharth** [comment]
@sammychinedu2ky Good work, Sam! Could you also share a recording on a mobile device size, to confirm if this is completely responsive?

**Comment by sammychinedu2ky** [comment]
> @sammychinedu2ky Good work, Sam! :) Could you also share a recording on a mobile device size, to confirm if this is completely responsive?

Yh sure let me do that

**Comment by sammychinedu2ky** [comment]
@paramsiddharth 
https://user-images.githubusercontent.com/36219292/201833214-88d7bf4a-07b8-40b9-83ef-f037c5b34c8a.mp4

**Comment by paramsiddharth** [comment]
> @paramsiddharth [user-images.githubusercontent.com/36219292/201833214-88d7bf4a-07b8-40b9-83ef-f037c5b34c8a.mp4](https://user-images.githubusercontent.com/36219292/201833214-88d7bf4a-07b8-40b9-83ef-f037c5b34c8a.mp4)

@sammychinedu2ky Looks amazing, Sam. :) Great work!

**Comment by sammychinedu2ky** [comment]
> > @paramsiddharth [user-images.githubusercontent.com/36219292/201833214-88d7bf4a-07b8-40b9-83ef-f037c5b34c8a.mp4](https://user-images.githubusercontent.com/36219292/201833214-88d7bf4a-07b8-40b9-83ef-f037c5b34c8a.mp4)
> 
> @sammychinedu2ky Looks amazing, Sam. :) Great work!

thanks 🥂

File: src/components/SearchReps.vue [hunk 1]
```diff
@@ -179,8 +185,9 @@ export default {
             postalCode: this.$route.params.postalCode || '',
             listVisible: false,
             isActive: false,
+            myHeight: this.$vuetify.breakpoint.mobile?false:"100vh"
```
Comment (paramsiddharth) [inline src/components/SearchReps.vue:188]:
```suggestion
            myHeight: this.$vuetify.breakpoint.mobile ? false : "100vh"
```

**Review by paramsiddharth — CHANGES_REQUESTED** [review]

**Review by paramsiddharth — APPROVED** [review]

**Pull request merged**